Products/ZenModel/MibOrganizer.py

```diff
@@ -235,7 +235,7 @@ def handleUploadedFile(self, REQUEST):
         if not mypath:
             mypath = '/'
         commandArgs = [binPath('zenmib'), 'run', savedMIBPath,
-                '--path=%s' % mypath]
+                '--path=%s' % mypath, '--mibdepsdir=%s' % zenPath("/var/ext/uploadedMIBs")]
```

Comment:
Please make `"/var/ext/uploadedMIBs"` a module level constant.

Why pick /var/ext/uploadedMIBs as the location instead of something more Zenoss specific?

Also, since `zenmib` has to be run in a container, does that path have an export to the local filesystem or will the user need to 'upload' the MIB file into that directory in the container?